ScoutHelper/Data/Turtle.json

```diff
@@ -1217,7 +1217,6 @@
 				"id": 1191,
 				"points": {
 					"905": {"x": " 8.50", "y": "33.30"},
-					"906": {"x": "12.80", "y": "21.40"},
```

Comment:
hmmm how strange. i wonder what happened to this point

Reply:
that's quite strange. good question
relating to this and your comment above, it was straight copy pasted from running the wiki's turtle code, so i'll take a look there for both issues as well as review the site real quick